plugin/trino-openlineage/src/test/java/io/trino/plugin/openlineage/MarquezServer.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.openlineage;
+
+import io.trino.testing.ResourcePresence;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.Closeable;
+import java.net.URI;
+import java.time.Duration;
+
+public class MarquezServer
+        implements Closeable
+{
+    private static final String MARQUEZ_CONFIG;
+    private static final int MARQUEZ_PORT;
+    private static final int MARQUEZ_ADMIN_PORT;
+    private static final String MARQUEZ_HEALTCHECK_API;
+    private static final String POSTGRES_HOST;
+    private static final String POSTGRES_DB;
+    private static final int POSTGRES_PORT;
+    private static final String POSTGRES_USER;
+    private static final String POSTGRES_PASSWORD;
+    private static final String LATEST_VERSION;
+
+    private Network network;
+
+    static {
+        MARQUEZ_CONFIG = "/opt/marquez/marquez.test.yaml";
+        MARQUEZ_PORT = 7000;
+        MARQUEZ_ADMIN_PORT = 7001;
+        MARQUEZ_HEALTCHECK_API = "/ping";
+        POSTGRES_HOST = "postgres";
+        POSTGRES_DB = "marquez";
+        POSTGRES_PORT = 5432;
+        POSTGRES_USER = "marquez";
+        POSTGRES_PASSWORD = "marquez";
+        LATEST_VERSION = "latest";
+    }
+
+    private final GenericContainer<?> dockerContainerAPI;
+    private final GenericContainer<?> dockerContainerPostgres;
+
+    public MarquezServer()
+    {
+        this(LATEST_VERSION);
+    }
```

Comment:
Change the default version to `0.46.0` instead.

Reply:
done